docs/en/index.md

```diff
@@ -1,66 +1,83 @@
-Quickstart
-==========
+kdyby/console
+=============
 
-This extension is here to provide integration of [Symfony Console](https://github.com/symfony/console) into Nette Framework.
+This extension provides integration of [Symfony Console](https://github.com/symfony/console) into [Nette Framework](https://www.nette.org).
+
+It allows you to create command-line commands directly within your application. These commands can be used for recurring tasks, as cronjobs, maintenances, imports and/or big things like sending newsletters.
 
 
 Installation
 -----------
 
-The best way to install Kdyby/Console is using  [Composer](http://getcomposer.org/):
+Fastest way is to use [Composer](http://getcomposer.org/) - run following command in your project root:
 
 ```sh
 $ composer require kdyby/console
 ```
 
-You can enable the extension using your neon config.
+Minimal configuration
+---------------------
 
-```yml
+First register new extension in your `config.neon`
+
+```
 extensions:
 	console: Kdyby\Console\DI\ConsoleExtension
 ```
 
-
-Minimal configuration
----------------------
-
-This extension creates new configuration section `console`, the absolute minimal configuration might look like this
+This creates new configuration section `console`, the absolute minimal configuration might look like this:
 
 ```yml
 console:
 	url: http://www.kdyby.org
 ```
 
-The `url` key specifies reference url that allows you to generate urls using Nette `UI\Presenter` in CLI (which is not possible otherwise). Another useful key is `commands` where you can register new commands. Look at the [Extending](#extending) part.
+The `url` key specifies reference url, that allows you to generate urls using Nette `UI\Presenter` in CLI (which is not possible otherwise).
```

Comment:
this is gonna need more rewriting, but that's definitely out of the scope of this PR